### PR TITLE
Devtools:-Removed unused CSS

### PR DESCRIPTION
**Pull request by Biki-das**
I tried to check out for the CSS bundle using the [eslint-plugin-css-modules](https://www.npmjs.com/package/eslint-plugin-css-modules), and ran it locally , seems like There are lot of unused styles and classes present .
I have attached the output below after running the lint command.

https://gist.github.com/Biki-das/647ceb7383b43cca9c8619e1dc33fe0d

All the shared.css files should not be touched i feel as they are being used randomly by some file 

Thoughts @hoxyq

### CODE REVIEW

File: packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js [hunk 1]
```diff
@@ -263,25 +263,19 @@ export default function InspectedElementWrapper(_: Props): React.Node {
         <div className={styles.SelectedComponentName}>
           <div
             className={
-              element.isStrictModeNonCompliant
-                ? styles.StrictModeNonCompliantComponent
-                : styles.Component
+              element.isStrictModeNonCompliant ? undefined : styles.Component
```
Comment (hoxyq) [inline packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js:266]: I believe there was a typo in https://github.com/facebook/react/pull/22886

so the right fix would be:
```suggestion
              element.isStrictModeNonCompliant ? styles.StrictModeNonCompliant : styles.Component
```

could you please update and verify that?

Other changes are fine, maybe we should explore supporting this eslint rule for `react-devtools-*` packages.

Comment (Biki-das) [reply packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js:266]: Ok sure, :-)

Comment (Biki-das) [reply packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js:266]: @hoxyq i thought of supporting the same, but the library is an unmaintained, the last update was 4 years ago.? is it fine to use such library ?

Comment (Biki-das) [reply packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js]: > 



> 

yes you are correct, it was a small typo!

Comment (hoxyq) [reply packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js]: What library are you referring to there? Sorry, didn't get that

Comment (hoxyq) [reply packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js]: Oh, you mean eslint plugin. Maybe there is a maintained one, which has the same purpose?

**Review by hoxyq — APPROVED** [review]

File: packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js (hunk 1 above)
Comment (Biki-das) [reply packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js]: ok @hoxyq meanwhile we can merge this one if its okay and let me research if there is something similar :-)

**Pull request merged**